packages/off-chain/scripts/main.ts

```diff
@@ -0,0 +1,95 @@
+// to be run on bacalhau via Docker container
+// file system is used to fetch the data from ../inputs/ directory
+import * as fs from "fs";
+import * as path from "path";
+import {successOutput, failedOutput} from "./outputSchema.js";
```

Comment:
These imports are not used as far as I understand. Are they necessary?

Reply:
Yes, currently it's not, but this imports are a defined schema or structure in which output should be stored.